### PR TITLE
Re-route CTest configuration page

### PR DESCRIPTION
**Pull request by williamjallen**
This PR is part of an ongoing effort to refactor our routes to eliminate *.php routes and create a coherent route structure.

### CODE REVIEW

File: resources/js/components/EditProject.vue [hunk 1]
```diff
@@ -17,7 +17,7 @@
       <a :href="$baseURL + '/project/' + cdash.project.Id + '/edit'">edit the project</a>
       <br>
       Click here to
-      <a :href="$baseURL + '/generateCTestConfig.php?projectid=' + cdash.project.Id">download the CTest configuration file</a>
+      <a :href="$baseURL + '/' + cdash.project.Id + '/ctest_configuration'">download the CTest configuration file</a>
```
Comment (josephsnyder) [inline resources/js/components/EditProject.vue:20]: Both of these URLs lack the `project`  folder to properly hit the API: 

image

File: resources/js/components/EditProject.vue [hunk 2]
```diff
@@ -1653,7 +1653,7 @@
                     </div>
                   </td>
                   <td>
-                    <a :href="$baseURL + '/generateCTestConfig.php?projectid=' + cdash.project.Id">
+                    <a :href="$baseURL + '/' + cdash.project.Id + '/ctest_configuration'">
```
Comment (josephsnyder) [inline resources/js/components/EditProject.vue:1656]: Same as above: image

**Review by josephsnyder — CHANGES_REQUESTED** [review]
The URL handling between `/ctest_configuration` and `generateCTestConfig.php` works great and the errors are handled well.  

But there are some broken links which just seemed to have dropped the `/project/` part of the expected URL.

File: resources/js/components/EditProject.vue (hunk 1 above)
Comment (williamjallen) [reply resources/js/components/EditProject.vue]: Good catch.  I tested by manually typing in the URL and forgot to check the links I changed.

**Review by josephsnyder — APPROVED** [review]
Great update!

**Pull request merged**